apps/meteor/client/components/message/toolbox/Toolbox.tsx

```diff
@@ -54,7 +59,15 @@ const Toolbox = ({ message }: ToolboxProps): ReactElement | null => {
 		);
 		const menuActions = await MessageAction.getButtons({ message, room, user, subscription, settings: mapSettings, chat }, context, 'menu');
 
-		return { message: messageActions, menu: menuActions };
+		const filteredActions =
```

Comment:
This filtering has some delay and can show up as a blinking icon to the final user. That's why in my opinion, we should filter the icon by label on the `MessageAction.ts` file

Reply:
I'm checking this behavior, maybe the root cause is not the filtering